readme.md

```diff
@@ -2,11 +2,16 @@
 
 [![Build Status](https://travis-ci.com/brumik/react-wooden-tree.svg?branch=master)](https://travis-ci.com/brumik/react-wooden-tree)
 
-A simple solution to display hierarchical structures (Tree View) while using React.
+The tree is bit more complex in the new version. You can use it with
+redux and of course without it. 
```

Comment:
This text highly depends on the old version's context, just simply explain that you have a redux-supported and a redux-free component. Without referring to the old version...imagine a complete stranger visiting your GH repo for the first time.